common/djangoapps/util/model_utils.py

```diff
@@ -152,6 +155,10 @@ def emit_setting_changed_event(user, db_table, setting_name, old_value, new_valu
         truncated_fields
     )
 
+    # Announce field change
+    USER_FIELD_CHANGED.send(sender=None, user=user, table=db_table, setting=setting_name,
```

Comment:
_old_value is deleted in line 75 of the post_save handler.  I can't be sure it will be there.